### PR TITLE
Craftedv2 transition guide

### PR DESCRIPTION
**Pull request by jeffbowman**
Comments or updates?

### CODE REVIEW

**Comment by sthesing** [comment]
Nope, I think it's fine as it is.

**Comment by jeffbowman** [comment]
If @jvdydev agrees, I'll merge it.  Thanks @sthesing for assisting on this!  It looks much better than my original attempt.

File: v1tov2transition-guide.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+# Transitioning your configuration
+
+Now for the hard part. This would be a great time to go read the [Getting Started
+Guide](https://github.com/SystemCrafters/crafted-emacs/docs/getting-started-guide.org),
+which explains the basic init and module structure of V2.
+
+## V1 config files to V2 config files
+
+Follow the advice in section [Bootstrapping Crafted
+Emacs](https://github.com/SystemCrafters/crafted-emacs/docs/getting-started-guide.org#bootstrapping-crafted-emacs)to
```
Comment (jvdydev) [inline v1tov2transition-guide.md:92]: Nit-picking here, but there is a missing space after the link.

File: v1tov2transition-guide.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+For example, replace:
+```elisp
+;; V1 config.el
+(require 'crafted-completion)
+```
+
+with:
+```elisp
+;; V2 init.el
+(require 'crafted-completion-packages) ;; in the packages section
```
Comment (jvdydev) [inline v1tov2transition-guide.md:166]: Idiomatic would be a single `;` after a line (again, nit-picking).

File: v1tov2transition-guide.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+like this:
+
+```elisp
+;; V1 config.el
+(crafted-package-install-package 'foo)
+```
+
+```elisp
+;; V2 init.el
+(add-to-list 'package-selected-packages 'foo) ;; in the packages section
```
Comment (jvdydev) [inline v1tov2transition-guide.md:183]: see the previous

**Review by jvdydev — APPROVED** [review]
Minor formatting nit-picks.
Otherwise looks good.

**Pull request merged**